test/functional/test_framework/util.py

```diff
@@ -326,6 +326,14 @@ def initialize_datadir(dirname, n, chain):
         os.makedirs(os.path.join(datadir, 'stdout'), exist_ok=True)
     return datadir
 
+def adjust_bitcoin_conf_for_pre_16_3(conf_file):
```

Comment:
```suggestion
def adjust_bitcoin_conf_for_pre_17(conf_file):
```

Sorry, this should say pre 17